src/formatDate.js

```diff
@@ -8,7 +9,41 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separador = fromFormat[3];
```

Comment:
The variable name `separador` is in Spanish, which is inconsistent with the rest of the code that is in English. It's best to maintain consistency in variable naming throughout the codebase. Consider renaming it to `fromSeparator` to match the English naming convention and improve code readability.